CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## v4.2.0 - 2025-02-28
+
+### Highlights
+ - The Kotlin bindings has received several API fixes in particular:
+   - AcmeChallenge was missing the target property
+   - proteusGetPrekeyFingerprint was missing
+ - The typescript bindings now correctly expose WireIdentity and X509Identity.
```

Comment:
Nitpick: `typescript` should probably be `Typescript` (see `Kotlin` above)

Also, although not a user-visible change, we may still want to mention that we moved to Rust 2024.

Reply:
Fixed